### PR TITLE
Feature/ssd

### LINKED ISSUE #8: Single Server Deployment
We need a single server deployment using docker-compose along with documentation on how to utilize it.

The single server deployment should not include the auth-service, as it currently requires an LDAP service provider to function. Because of this, we will need to include documentation on how to send authentication to layer1-service. Additionally, we will also exclude the store-service, since functionality with minio has yet to be tested and verified.

### PR DESCRIPTION
**Pull request by HuiJun**
Adding a docker-compose which includes all required services.

### CODE REVIEW

**Comment by HuiJun** [comment]
I believe we still need to add ldap groups to the initial cluster.trig file.

File: docs/mms5-layer1-service/index.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
+MMS 5 Layer 1
```
Comment (dlamoris) [inline docs/mms5-layer1-service/index.rst]: remove the docs folder, it'll be generated on commits to develop

File: docker-compose/env/mms5-auth.env [hunk 2]
```diff
@@ -0,0 +1,9 @@
+LDAP_LOCATION=ldap://openldap-server:1389
+LDAP_BASE=dc=example,dc=org
+LDAP_USER_PATTERN=cn=%s,ou=users
+LDAP_USER_NAMESPACE=ldap/user/
+LDAP_GROUP_SEARCH_FILTER=(&(objectclass=group)(member=%s)(|(%s)))
+LDAP_GROUP_ATTRIBUTE=cn
+LDAP_GROUP_NAMESPACE=ldap/group/
+LDAP_GROUP_STORE_URI=http://quad-server:3030/ds/sparql
+LDAP_GROUP_STORE_CONTEXT=http://layer1-service
```
Comment (dlamoris) [inline docker-compose/env/mms5-auth.env]: the LDAP_GROUP_STORE_CONTEXT actually needs a / at the end here

**Comment by dlamoris** [comment]
pushed an update to some env vars and the trig file, not sure if the created users are already part of the ldap group though, the token doesn't include the group for either user

**Review by dlamoris — APPROVED** [review]

**Pull request merged**